.circleci/config.yml

```diff
@@ -144,6 +144,23 @@ jobs:
 
       - run: bundle exec fastlane lint_source
 
+  "Execute modules load up tests":
+    environment:
+      LC_ALL: C.UTF-8
+      LANG: C.UTF-8
+    docker:
+      - image: circleci/ruby:2.3
+    shell: /bin/bash --login -eo pipefail
+    steps:
+      - checkout
+
+      - run:
+          name: Setup Build
+          command: |
+            gem install $(echo $(gem build fastlane.gemspec) | sed 's/.*File: //')
```

Comment:
What does this do?

Reply:
It builds gem from gemspec and installs it right away

Reply:
The wonders of shell/bash stuff...

(Might want to add `Build gem from fastlane.gemspec and install locally` or similar as a comment above)